docs/docs/application-basics.md

```diff
@@ -4,12 +4,13 @@ title: Application Basics
 
 # Application Basics
 
-Applications are an integral concept in Marathon. Each application typically represents a long running service, of which there would be many instances running on multiple hosts.
+Applications are an integral concept in Marathon. <!-- are they a concept? if so, what more can we say about this concept? --> Each application typically represents a long-running service, of which there would be many instances running on multiple hosts.
```

Comment:
Perhaps there is a better term? An application definition describes everything to start and maintain tasks.